### DISCUSSION TITLE
A program like mkvtoolnix but for MP4 files?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [staxrip/staxrip#476] [Q&A] [answered]
A program like **mkvtoolnix** but for **MP4** files that let you modify the head info without re-encoding?

**Comment by ZeroCool22** [comment, marked as answer]
I found what i was looking for.

👉 **XMedia Recode 64bit**

**Reply by JJKylee** [reply]
Well, XMedia Recode(a.k.a. XMR) is just one of the many GUIs for **ffmpeg**, _not_ MP4Box, the ultimate MP4 muxer/demuxer.

![XMedia-Recode_3 5 2 7_about-popup_20210205](https://user-images.githubusercontent.com/3157518/107033789-a8231d80-676a-11eb-831d-8a903626df6f.jpg)

It uses the media tools shown above, but it demuxes/muxes via FFmpeg _only_.

So, technically, your answer is _not precise_.
If XMR was a toolset with GUI for MP4Box, StaxRip would be another. StaxRip can also edit MP4 metadata/tags _without re-encoding_.

AFAIK, there's no MP4Box counterpart for mkvtoolnix as of now. (There used to be some, but IMHO, they are all outdated.)

* Just out of curiosity: why do you keep asking questions and adopting your own (sometimes incorrect) answers so quickly? If you'd like to share what you know, you can always go to Wiki. Or if you asked a question, you could have used more patience. 😑
